docs/terms/profiler.md

```diff
@@ -1,15 +1,163 @@
+## Overview
 
+### Definition
 
-NOTES: Everything under here is temporary, for use in writing the detailed page.
-----------
-Conceptually, each Profiler is a checklist of questions.
\ No newline at end of file
+A Profiler generates <TechnicalTag relative="../" tag="metric" text="Metrics" /> and candidate <TechnicalTag relative="../" tag="expectation" text="Expectations" /> from data.
+
+### Features and promises
+
+A Profiler makes it possible to quickly create a starting point for generating Expectations about a dataset. For example, during the [Getting Started Tutorial](../tutorials/getting_started/tutorial_overview.md), Great Expectations uses the `UserConfigurableProfiler` to demonstrate important features of Expectations by creating and validating an <TechnicalTag relative="../" tag="expectation_suite" text="Expectation Suite" /> that has several kinds of Expectations built from a small sample of data. A Profiler is also critical to generating the Expectation Suites used during profiling.
+
+There are several Profilers included with Great Expectations; conceptually, each Profiler is a checklist of questions which will generate an Expectation Suite when asked of a Batch of data.
+
+### Relationship to other objects
+
+A Profiler builds an Expectation Suite from one or more Data Assets. It usually also <TechnicalTag relative="../" tag="validation" text="Validates" /> the data against the newly-generated Expectation Suite to return a <TechnicalTag relative="../" tag="validation_result" text="Validation Result" />.
+
+## Use cases
+
+<CreateHeader/>
+
+Profilers come into use when it is time to create Expectations for your project.  At this point in your workflow you can create a new Profiler, or use an existing one to generate Expectations from a <TechnicalTag relative="../" tag="batch" text="Batch" /> of data.
+
+For details on how to create a customized Profiler, see our guide on [how to create a new expectation suite using rule based Profilers](../guides/expectations/advanced/how_to_create_a_new_expectation_suite_using_rule_based_profilers.md).
+
+For instructions on how to use a Profiler to generate Expectations from data, see our guide on [how to create and edit Expectations with a Profiler](../guides/expectations/how_to_create_and_edit_expectations_with_a_profiler.md).
+
+## Features
+
+### Multiple types of Profilers available
+
+There are multiple types of Profilers built in to Great Expectations.  Below is a list with overviews of each one.  For more information, you can view their docstrings and source code in the `great_expectations\profile` [folder on our GitHub](https://github.com/great-expectations/great_expectations/tree/develop/great_expectations/profile).
+
+#### UserConfigurableProfiler
+
+The `UserConfigurableProfiler` is used to build an Expectation Suite from a Dataset. The Expectations built are strict - they can be used to determine whether two tables are the same.
```

Comment:
Capital D Dataset was a V2 entity, lowercase is a bit clearer.

```suggestion
The `UserConfigurableProfiler` is used to build an Expectation Suite from a dataset. The Expectations built are strict - they can be used to determine whether two tables are the same.
```

Reply:
I wouldn't say "they can be used to determine whether two tables are the same." - this is too strong. We've said things in the past that it can be used as a proxy for identicality. As Will mentions, it might make sense to flesh this out and explain that in order to make this determination, you profile on dataset, then validate another.